include/ignition/gui/qml/Chart.qml

```diff
@@ -561,13 +625,11 @@ Rectangle {
       onClicked: {
         main.clicked(chartID);
       }
-      onDoubleClicked: chart.zoomReset();
```

Comment:
Why is this removed?

Reply:
i used that just for testing :D 
not part of the code